enterprise/releases/2024-10-0.mdx

```diff
@@ -0,0 +1,92 @@
+  - After upgrading the installation will still be able to read from the object storage for old logs, but no longer write to it. After 7 days have passed (the log retention time) you can remove all object storage related configuration.
+
+## Features
+
+- Index Advisor
+  - Add support for "pg_bigm", "cube" and "earthdistance" extensions <!-- 3953 -->
+  - Avoid Internal Error for tables that had no scans/query activity in the last 7 days <!-- 3956 -->
+  - Avoid errors when unsupported index types/data types are referenced by a table <!-- 3954 -->
+  - Add support for "ltree" extension <!-- 3957 -->
+  - Add ability to filter out acknowledged state insights <!--3943  -->
+  - Fix the count in "QUERIES" column in insights list <!-- 4288 -->
+    - In the Missing Indexes tab in the Index Advisor page, the column "QUERIES" was showing the number of missing indexes in that insight (table) previously. Now this column shows the number of unique affected queries due to these missing indexes as intended.
+  - Show affected index count of each table in insights list <!-- 4289 -->
+  - Recommend removing unused expression indexes in Unused Index alert <!-- 4185 -->
+    - Previously expression indexes were excluded because they produce column statistics that the query planner can benefit from. But this was confusing for users who knew some indexes weren't being queried, and yet we didn't recommend deleting them. You may want to additionally `CREATE STATISTICS` for the expression when dropping an expression index if you notice worse query plans.
+- Automated EXPLAIN and EXPLAIN Plans
+  - Automated EXPLAIN: Add a better support for handling new lines with the text log format <!-- 3937 -->
+  - View EXPLAIN Source: Hide "Async Capable" field when converting from JSON to text format <!-- 3946 -->
+  - Support all Aggregate nodes with text format explain <!-- 4020 -->
+    - Previously, only the plain Aggregate with text format explain was supported. With this change, we support GroupAggregate, HashAggregate, and MixedAggregate better.
+  - Revise EXPLAIN details layout <!-- 4174 -->
+- Update bundled collector from 0.58.0 to 0.61.0 <!-- XXX -->
+  - See the [collector changelog](XXX) for the full details
+- Raise connection tracing retention time from 24 hours to 7 days with new packed stats format
+- Alert Policy: Enhanced Email notifications <!-- 4082 -->
+  - Email notifications are now sent by default for all critical severity issues. Previously, some critical issues did not trigger email notifications due to an oversight. This has been corrected.\nSeveral checks were missing from the Alert Policy page, which prevented email notifications for those checks. This update includes all checks on the Alert Policy page, making it possible to enable email notifications for every critical severity issue.
```

Comment:
```suggestion
  - Email notifications are now sent by default for all critical severity issues. Previously, some critical issues did not trigger email notifications due to an oversight. This has been corrected. Several checks were missing from the Alert Policy page, which prevented email notifications for those checks. This update includes all checks on the Alert Policy page, making it possible to enable email notifications for every critical severity issue.
```
Is this a literal "\n"? That seems like it should not be there.

Reply:
Err, yeah, that crept in from copying this from the changelog JSON file.